modules/tfc-oidc/main.tf

```
@@ -0,0 +1,60 @@
+# Copyright (c) HashiCorp, Inc.
+
+locals {
+  attribute_condition = var.attribute_condition == "" ? "assertion.sub.startsWith(\"organization:${var.tfc_organization_name}:project:${var.tfc_project_name}:workspace:${var.tfc_workspace_name}\")" : var.attribute_condition
+}
+
+# Enables the required services in the project.
+# https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_service
+resource "google_project_service" "services" {
+  count   = length(var.service_list)
+  service = var.service_list[count.index]
```

Got this error here without adding the project_id: `Error: project: required field is not set`, also dependent service errors when destroying.

```suggestion
  count              = length(var.service_list)
  project            = var.project_id
  service            = var.service_list[count.index]
  disable_on_destroy = false
```